content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md

```diff
@@ -36,6 +36,11 @@ Other than that, there are some minor bugfixes which you can find in the list of
 With PrestaShop 9, there's certainly more going on. Work on Symfony Layout, API, cleaning: there is a lot of progress, but the team still needs some time to better present the results. Regarding the new API, which is being implemented into PrestaShop 9, [some interesting developments are happening](https://github.com/PrestaShop/PrestaShop/pull/34171), and we will be sharing more details about them soon!
 A great time to learn more about the new API is the PrestaShop Developer Conference, where [@joelievre](https://github.com/jolelievre) will be [presenting the new API](https://events.prestashop.com/prestashop-developer-conference/en/sessions?searchtext=api&page=1) to the audience.
 
+
+### Welcome to Dominik Shaim, new PrestaShop committer
+
+In November we welcomed a new PrestaShop committer - [@ShaiMagal](Dominik Shaim), community member from Czech Republic and founder of [OpenServis](https://www.openservis.cz/), company specialized in the PrestaShop hosting and support. Dominik is experienced developer who work with PrestaShop since 2012, and he's been contributing to the project for a while now. We're happy to have him on board, and we're looking forward to working with him! You can see a list of all committers on [the PrestaShop project website](https://www.prestashop-project.org/project-organization/people-and-roles/#committers).
```

Comment:
```suggestion
In November we welcomed a new PrestaShop committer - [@ShaiMagal](Dominik Shaim), community member from Czech Republic and founder of [OpenServis](https://www.openservis.cz/), company specialized in the PrestaShop hosting and support. Dominik is an experienced developer who works with PrestaShop since 2012, and he's been contributing to the project for a while now. We're happy to have him on board, and we're looking forward to working with him! You can see a list of all committers on [the PrestaShop project website](https://www.prestashop-project.org/project-organization/people-and-roles/#committers).
```